src/main/java/org/apache/sling/resourceresolver/impl/ResourceResolverFactoryActivator.java

```diff
@@ -304,6 +312,25 @@ protected void activate(final BundleContext bundleContext, final ResourceResolve
             this.observationPaths[i] = new Path(paths[i]);
         }
 
+        // optimize alias path allow list
+        this.aliasPathAllowList = null;
+        String[] aliasPathPrefix = config.resource_resolver_optimize_alias_allowlist();
+        if ( aliasPathPrefix != null ) {
+            final List<String> prefixList = new ArrayList<>();
+            for(final String value : aliasPathPrefix) {
+                if ( value.trim().length() > 0 ) {
```

Comment:
You used `value.trim()` 3 times, I'd suggest to extract it into a local variable.